jumpcloud-ADMU/Powershell/Start-Migration.ps1

```diff
@@ -660,13 +896,17 @@ Function Get-WindowsDrive {
   .LINK
      https://gallery.technet.microsoft.com/scriptcenter/Write-ToLog-PowerShell-999c32d0
   #>
+# Set a global parameter for debug logging
+$Script:AdminDebug = $false
```

Comment:
These should be moved to Start-Migration - lets keep our code within in functions rather than these random locations